treeherder/model/migrations/0054_duplicate_failure_classifications.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-20 15:05
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+# Original SQL:
+#
+# BEGIN;
+# --
+# -- Create model TextLogErrorMatch
+# --
+# CREATE TABLE `text_log_error_match` (`id` bigint AUTO_INCREMENT NOT NULL PRIMARY KEY, `score` numeric(3, 2) NULL, `classified_failure_id` bigint NOT NULL, `matcher_id` integer NOT NULL, `text_log_error_id` bigint NOT NULL);
+# --
+# -- Create model TextLogErrorMetadata
+# --
+# CREATE TABLE `text_log_error_metadata` (`best_is_verified` bool NOT NULL, `best_classification_id` bigint NULL, `failure_line_id` bigint NULL UNIQUE, `text_log_error_id` bigint NOT NULL PRIMARY KEY);
+# --
+# -- Add field text_log_errors to classifiedfailure
+# --
+# --
+# -- Alter unique_together for textlogerrormatch (1 constraint(s))
+# --
+# ALTER TABLE `text_log_error_match` ADD CONSTRAINT `text_log_error_match_text_log_error_id_192a8c4c_uniq` UNIQUE (`text_log_error_id`, `classified_failure_id`, `matcher_id`);
+# ALTER TABLE `text_log_error_match` ADD CONSTRAINT `text_log_classified_failure_id_1b38b499_fk_classified_failure_id` FOREIGN KEY (`classified_failure_id`) REFERENCES `classified_failure` (`id`);
+# ALTER TABLE `text_log_error_match` ADD CONSTRAINT `text_log_error_match_matcher_id_9b9dbb12_fk_matcher_id` FOREIGN KEY (`matcher_id`) REFERENCES `matcher` (`id`);
+# ALTER TABLE `text_log_error_match` ADD CONSTRAINT `text_log_error_m_text_log_error_id_09109841_fk_text_log_error_id` FOREIGN KEY (`text_log_error_id`) REFERENCES `text_log_error` (`id`);
+# ALTER TABLE `text_log_error_metadata` ADD CONSTRAINT `text_lo_best_classification_id_54f54ccb_fk_classified_failure_id` FOREIGN KEY (`best_classification_id`) REFERENCES `classified_failure` (`id`);
+# ALTER TABLE `text_log_error_metadata` ADD CONSTRAINT `text_log_error_metad_failure_line_id_2b5bd526_fk_failure_line_id` FOREIGN KEY (`failure_line_id`) REFERENCES `failure_line` (`id`);
+# ALTER TABLE `text_log_error_metadata` ADD CONSTRAINT `text_log_error_m_text_log_error_id_c2600712_fk_text_log_error_id` FOREIGN KEY (`text_log_error_id`) REFERENCES `text_log_error` (`id`);
+# COMMIT;
```

Comment:
Is this actually true? The autogenerated migration has foreign key checks enabled and generates the table in multiple statements rather than in one per table. So the two are different. I don't know if it's a meaningful difference from a performance point of view.